packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -542,12 +555,13 @@ export async function processCypressTestRecording(
               }
             });
 
-            assert(beginPoint !== null, `Missing "step:start" annotation for test event`, {
+            assert(stepStartPoint !== null, `Missing "step:start" annotation for test event`, {
               id,
-              isChaiAssertion,
             });
-
-            assert(viewSourceTimeStampedPoint !== null, `Missing annotation for test event`, {
+            assert(stepEnqueuePoint !== null, `Missing "step:end" annotation for test event`, {
```

Comment:
Should this assert on `stepEndPoint` instead?

Reply:
On my phone so I can't see easily but will this error on chai assertions that lack r queue? If we're not sure, I can check in the morning

Reply:
I am so confused by these inconsistencies in the data 😞 

Yes, enqueue is required as of this change because we need it for focus. I did not think it would be missing entirely for "chai assertions" just that it wasn't the point we wanted to use for viewing the source code.